src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -1175,10 +1176,13 @@ public synchronized void afterMerge(OnGoingMerge merge) {
     }
 
 
-    private void commitIndexWriter(IndexWriter writer, long translogId) throws IOException {
+    private void commitIndexWriter(IndexWriter writer, Translog.TranslogGeneration commit) throws IOException {
```

Comment:
nah that won't work since we not always use the member but I can just pass the translog instead